find_me_a_sandwich/app/interactors/search_locu_venues.rb

```diff
@@ -9,7 +9,7 @@ def call
 
   private
 
-  def build_location(location, location_data)
+  def build_location(_location, location_data)
```

Comment:
Looks like this is leftover from a previous iteration.  Feel free to refactor the first param out.